README.Rmd

```diff
@@ -25,10 +25,12 @@ knitr::opts_chunk$set(
 
 - `get_resource()` extracts a single resource from an open dataset by resource id
 - `get_dataset()` extracts multiple resources from an open dataset by dataset name
+- `list_datasets()` returns the names of all available datasets 
+- `list_resources()` returns information on all resources within an open dataset by dataset name
 
 For extracting metadata and search functionality, we recommend using the [ckanr package](https://docs.ropensci.org/ckanr/). 
```

Comment:
Can this line be removed? Wasn't there an issue with the ckanr package?
```suggestion
```

Maybe we add a *See also* section to the bottom of the readme and link to ckanr there.